source/includes/_platform-payments.md

```diff
@@ -158,198 +153,179 @@ Updates a specific set of stored account details. This will trigger an AML check
-| officerList.items.dateOfBirth               | string | ISO-8601 format date                           | required |
-| officerList.items.firstName                 | string |                                                | required |
-| officerList.items.lastName                  | string |                                                | required |
-| officerList.items.resignedOn                | string |                                                | |
-| officerList.items.officerRole               | string |                                                | required |
-| officerList.items.emailAddress              | string |                                                | required |
-| officerList.items.phoneNumber               | string |                                                | required |
-| officerList.items.percentageOwnership       | number |                                                | required |
-| officerList.items.address                   | ref    |                                                | required |
-| officerList.items.address.addressLine1      | string | Address Line 1                                 | required |
-| officerList.items.address.addressLine2      | string | Address Line 2                                 | required |
-| officerList.items.address.careOf            | string | Care Of                                        | required |
-| officerList.items.address.country           | string | Country                                        | required |
-| officerList.items.address.locality          | string | Locality                                       | required |
-| officerList.items.address.poBox             | string | Po Box                                         | required |
-| officerList.items.address.postalCode        | string | Postal Code                                    | required |
-| officerList.items.address.region            | string | Region                                         | required |
-
-## `POST /platformApi/wallet`
-```http
+| Name              | Type   | Description                                        | Required |
+| ----------------- | ------ | -------------------------------------------------- | -------- |
+| ukCompany.number  | string | The Company Number registered with Companies House.| required |
+| ukCompany.name    | string | Company Name                                       | required |
 
-POST /platformApi/wallet HTTP/1.1
+### Response
+| Name              | Type   | Description                                        |
+| ----------------- | ------ | -------------------------------------------------- |
+| partyId           | string                                                      |
+| ukCompany.number  | string | The Company Number registered with Companies House.|
+| ukCompany.name    | string | Company Name                                       |
```

Comment:
Will we need a status of sorts? Whether that would be for KYB but generally since we have an asynchronous process to complete details and send to Modulr...

Reply:
We spoke about this, and then didn't make a note of it.

I've added a 'status', which will either be 'PENDING' or 'REGISTERED' and have detailed a webhook called 'COMPANY_REGISTRATION_UPDATE'.